app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -1,18 +1,17 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
 
 import { Plus } from "@bigbinary/neeto-icons";
-
 import { Button } from "neetoui/v2";
 import { Container, Header } from "neetoui/v2/layouts";
-import constants from "./constants"
 
 import ContactsList from "./ContactsList";
+import NewContact from "./Create/Contact";
```

Comment:
Why is the file structure `Create/Contact`? 
Can't we just keep it as `Create.jsx`